ui/apps/ui/src/app/pages/guidelines-page/guideline-detail-page.component.ts

```diff
@@ -0,0 +1,146 @@
+import { Component, OnInit } from '@angular/core';
+import { IResult, ITag } from '@collections/repositories/types';
+import { GuidelinesService } from './guidelines.service';
+import { ActivatedRoute } from '@angular/router';
+import { guidelinesAdapter } from '@collections/data/guidelines/adapter.data';
+import isArray from 'lodash-es/isArray';
+import { UntilDestroy, untilDestroyed } from '@ngneat/until-destroy';
+import moment from 'moment';
+
+@UntilDestroy()
+@Component({
+  selector: 'ess-guideline-detail-page',
+  templateUrl: './guideline-detail-page.component.html',
+  styleUrls: ['./guideline-detail-page.component.scss'],
+})
+export class GuidelineDetailPageComponent implements OnInit {
+  guideline?: IResult;
+  currentTab = 'about';
+  isArray = isArray;
+  authorNames?: string[];
+  authorNamesTg?: string[];
+  authorTypes?: string[];
+  description?: string[];
+  doi?: string[];
+  eoscGuidelineType?: string;
+  eoscIntegrationOptions?: string[];
+  id?: string;
+  publicationYear?: number;
+  rightId?: string[];
+  rightTitle?: string[];
+  rightUri?: string[];
+  title?: string[];
+  typeInfo?: string[];
+  updateAt?: string;
+  _version_?: number;
+  publicationDate?: string;
+  type?: string;
+  status?: string;
+  facets?: object;
+  domain?: string[];
+  identifierType?: string[];
+  catalogue?: string[];
+  givenName?: string[];
+  familyName?: string[];
+  affiliation?: string[];
+  affiliationIdentifier?: string[];
+  nameIdentifier?: string[];
+  provider?: string[];
+  eoscRelatedStandards?: string[];
+
+  statusesDictionary = new Map([
+    ['ir_status-abandoned', 'Abandoned'],
+    ['ir_status-accepted', 'Accepted'],
+    ['ir_status-candidate', 'Candidate'],
+    ['ir_status-proposed', 'Proposed'],
+    ['ir_status-consultation', 'Consultation'],
+    ['ir_status-deprecated', 'Deprecated'],
+    ['ir_status-on_hold', 'On Hold'],
+    ['ir_status-operating', 'Operating'],
+    ['ir_status-rejected', 'Rejected'],
+    ['ir_status-withdrawn', 'Withdrawn'],
+  ]);
+
+  eoscGuidelineTypeDictionary = new Map([
+    [
+      'ir_eosc_guideline_type-eosc_core_interoperability_guideline',
+      'EOSC-Core Interoperability Guideline',
+    ],
+    [
+      'ir_eosc_guideline_type-eosc_exchange_interoperability_guideline_horizontal',
+      'EOSC-Exchange Interoperability Guideline (Horizontal)',
+    ],
+    [
+      'ir_eosc_guideline_type-eosc_exchange_interoperability_guideline_thematic',
+      'EOSC-Exchange Interoperability Guideline (Thematic)',
+    ],
+    ['ir_eosc_guideline_type-operational_baseline', 'Operational Baseline'],
+  ]);
+
+  constructor(
+    private guidelinesService: GuidelinesService,
+    private route: ActivatedRoute
+  ) {}
+
+  ngOnInit(): void {
+    this.getItem();
+  }
+
+  getItem(): void {
+    const id = this.route.snapshot.paramMap.get('guidelineId') || 1;
+    this.guidelinesService
+      .get$(id)
+      .pipe(untilDestroyed(this))
```

Comment:
untilDestroy is redundant here - this is not a long living observable

Reply:
Maybe now it is ok?